### ISSUE TITLE
Proposal: make pgddl installation "cloud-friendly"

### ISSUE THREAD

**Issue by ayuryshev** [lacanoid/pgddl#20]
# Problem

Normal postgresql way to install extensions is not possible with cloud-providers offerings like Azure PostgreSQL Database or Google CloudSQL for PostgreSQL (I did not checked but it must be similar for Amazon too).

List of extensions possible to install is determined by cloud provider:
- Azure: https://docs.microsoft.com/en-us/azure/postgresql/concepts-extensions
- Google: https://cloud.google.com/sql/docs/postgres/extensions#postgresql-extensions-supported-by-cloud-sql

In theory it's possible to write to provider support: "please make available extension XXX"   but chances are not big.

# Proposed solutions

A. Just describe procedure to create SQL-script without `make install` and so on. Section in README.md or wiki.

B.  Maybe option `make cloud-install` with env-parameters for connection : host, database, port  

C. Maybe separate script

**Comment by lacanoid** [comment]
The extension is SQL only, perhaps this can be of some use of persuading cloud providers to allow it.

When you build the extension, it builds the script which you can source into psql like a normal SQL file. It is not really necessary to install it. This might cause some problems when upgrading postgres to new version, as system catalogs change sometimes...

**Comment by ayuryshev** [comment]
Yes.
We work with Azure and I installed it just as collection of functions.

We are thinking about migration from 11.6 (version in Azure Basic Server) to PostgreSQL 12.x - and I totally forgot how exactly I produced script previously. 

I suspect that I run `[ENV_VARS = .... ] make ` but I'm not sure about it.

Maybe just line in README.md would be OK.
And `make cloud-install` - I'm not confident with PGXN, does it allows have another Makefile goals?
